app_vue/src/styles/telusUi.js

```diff
@@ -0,0 +1,4 @@
+export const telusUi = {
+    green : "#2B8000",
+    purple: "#4B286D"
+};
```

Comment:
Nice way of allowing us to refer to colour variables in the javascript side!

Should we rename these colours to `telusGreen` and `telusPurple`? theres already an existing `green` colour in `app_vue/src/assets/stylesheets/_variables.scss` and i suppose it could get confusing eventually